cookbooks/aws-parallelcluster-slurm/templates/default/slurm/login_nodes_daemon_service/login_nodes_daemon_config.json.erb

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I suggest to keep the directory structure consistent with the rest of the project.
I suggest to replace `templates/default/slurm/login_nodes_daemon_service` with `templates/default/slurm/login`

Reply:
Okay. Done!